### ISSUE TITLE
MSAL integration

### ISSUE THREAD

**Issue by ukanth** [msintuneappsdk/Taskr-Sample-Intune-Android-App#5]
Hello,

Currently this example uses ADAL for intune MAM sdk. Are there any plans to use MSAL ?

**Comment by ukanth** [comment]
I see 6.3 with MSAL support. But this sample still used ADAL.

**Comment by viveknegi51288** [comment]
Hello @meghandaly , Could you please help me to integrate MSAL library.?

As I have develped Xamarin forms application using MSAL library and able to get access token for ios, but I need to registered the user in intune using MSAL library. Please help

**Comment by Innova133** [comment]
Yes, as all the documentation recommends us using MSAL, can we get a sample application that integrates the Intune SDK using MSAL for authentication?

**Comment by meghandaly** [comment]
The app has been updated to feature MSAL integration 👍
